client/src/pages/DashboardPage.jsx

```diff
@@ -19,6 +24,8 @@ const useStyles = makeStyles(() => ({
   },
 }));
 
+let socket;
```

Comment:
As it doesn't seem to really get reassigned or initialized with other parameters, could we go as far as `const socket = socketIOCLient(SOCKET_URL);` here and connect/disconnect it in useEffect?

Reply:
Looked together over a meet call